src/normative_only_index.rst

```diff
@@ -11,9 +11,9 @@ Normative CellML 2.0 Specification
 This document is the normative version of the CellML Specification, defining the CellML syntax and the rules by which it should be used.
 It is intended primarily for the developers of software tools which directly consume CellML syntax.
 
-Users of CellML models should read the informative version of the CellML Specification which is available at https://cellml.org/specifications/cellml_2.0.
+Users of CellML models may prefer the informative version of the CellML Specification, which is available at https://cellml.org/specifications/cellml_2.0.
```

Comment:
These should be added to the named reference lists ... I will do that in another PR.